model/src/guild/mod.rs

```diff
@@ -51,10 +53,19 @@ use crate::{
 use std::collections::HashMap;
 
 #[cfg(feature = "serde-support")]
-fn default_max_presences() -> u64 {
+pub(crate) fn default_max_presences() -> u64 {
     5000
 }
 
+#[cfg(feature = "serde-support")]
+pub(crate) fn deserialize_max_presences<'de, D>(deserializer: D) -> Result<u64, D::Error>
+where
+    D: serde::de::Deserializer<'de>,
+{
+    use serde::Deserialize as _;
```

Comment:
Is this import needed? It looks like it's just being imported into the void 👻

Reply:
It is needed to import the trait on `u64` so I can use it directly.